mozilla-release/mobile/android/app/src/photon/java/org/mozilla/gecko/toolbar/ToolbarDisplayLayout.java

```diff
@@ -601,23 +604,6 @@ void destroy() {
     }
 
     /* Cliqz start */
-    // stop scrolling title, it ellipses form the end.change the padding if we display url
-     private class TextChangeListener implements TextWatcher {
```

Comment:
you shouldn't remove firefox codebase, just comment it with cliqz start and cliqz end block.
the original code of the class was
```
    private class TextChangeListener implements TextWatcher {
        @Override
        public void afterTextChanged(Editable text) {
            scrollTitle();
        }

        @Override
        public void beforeTextChanged(CharSequence s, int start, int count, int after) { }

        @Override
        public void onTextChanged(CharSequence s, int start, int before, int count) { }
    }
```

Reply:
Thanks Moaz. Cause of the 'Cliqz Start' block I misunderstood that we probably added it.
Changing it soon!